README.md

```diff
@@ -1,15 +1,15 @@
-# GraphQL Custom Scalars Specification Contribution Guide
+# GraphQL scalars Specification Contribution Guide
 
-Thanks for contributing to GraphQL Scalars.
+Thanks for contributing to GraphQL scalars.
```

Comment:
`GraphQL Scalars` would be the title of the project/repository, so title case is correct.